neo/Consensus/ConsensusService.cs

```diff
@@ -195,28 +195,7 @@ private void OnChangeViewReceived(ConsensusPayload payload, ChangeView message)
             // additional recovery message response.
             if (!knownHashes.Add(payload.Hash)) return;
             if (message.NewViewNumber <= context.ViewNumber)
```

Comment:
Could we have a log here: 
Log($"{nameof(OnChangeViewReceived)}: height={payload.BlockIndex} view={message.ViewNumber} index={payload.ValidatorIndex} nv={message.NewViewNumber} - Going to activate {nameof(OnRecoveryRequestReceived)}");

Reply:
```cs
             if (message.NewViewNumber <= context.ViewNumber)
+            {
+                Log($"{nameof(OnChangeViewReceived)}: nv={message.NewViewNumber} - calling {nameof(OnRecoveryRequestReceived)}");
                 OnRecoveryRequestReceived(payload);
+            }
```